skore/src/skore/sklearn/_plot/metrics/confusion_matrix.py

```diff
@@ -1,12 +1,11 @@
 import matplotlib.pyplot as plt
 import numpy as np
-from sklearn.metrics import confusion_matrix as sklearn_confusion_matrix
 
-from skore.sklearn._plot.base import Display
 from skore.sklearn._plot.style import StyleDisplayMixin
+from skore.sklearn._plot.utils import HelpDisplayMixin
```

Comment:
We should not decorate `__init__` but the `plot` method (cf. `RocCurveDisplay` for a concrete example)